sql-aggregates/on-the-map.sql

```diff
@@ -0,0 +1,13 @@
+select "countries"."name" as "countryName",
+  count("cityId") as "TotalNumberCities"
+from "countries"
+join "cities"  on "countries"."countryId" = "cities"."countryId"
+group by  "countries"."countryId", "countries"."name";
```

Comment:
Grouping just by the countryId is all that is needed. The countryId will be unique for every country.